backend/deploy.js

```diff
@@ -1,11 +1,12 @@
 var crypter = require("../util/crypter.js");
 var spawn = require('child_process').spawn;
 var output = require('../util/output');
+var socketHandler = require('../util/socketHandler.js');
 
-exports.deployPages = function (socket, data) {
+exports.deployPages = function (socket, data, callback) {
```

Comment:
@pri22296 I agree with your query but then those are not pure function how will write test case if the function doesn't return anything. That's why I added a callback. if you have any workaround to test this kind of function please suggest me.